Documentation/ApiOverview/Authentication/_CSRFlikeRequestTokenHandling/_MyController.php

```diff
@@ -1,17 +1,21 @@
 <?php
 
 use TYPO3\CMS\Core\Security\RequestToken;
-use TYPO3\CMS\Fluid\View\StandaloneView;
+use TYPO3\CMS\Core\View\ViewFactoryData;
+use TYPO3\CMS\Core\View\ViewFactoryInterface;
 
 final class MyController
 {
-    private StandaloneView $view;
+    public function __construct(
+        private readonly ViewFactoryInterface $viewFactory,
+    ) {}
 
     public function showFormAction()
     {
+        $view = $this->viewFactory->create(new ViewFactoryData(/* ... */));
```

Comment:
Either set the arguments as example:

```php
    new ViewFactoryData(
        templateRootPaths: ['EXT:myExt/Resources/Private/Templates'],
        partialRootPaths: ['EXT:myExt/Resources/Private/Partials'],
        layoutRootPaths: ['EXT:myExt/Resources/Private/Layouts'],
        request: $request,
    );
```

or link in text to the chapter where ViewFactory/ViewFactoryData is explained. This eases the adoption for a developer.

Reply:
Feel free to make a follow up. That would also mean we have to determine where we get the request from which is not available yet etc. It introduces a whole new level of complexity.